osu.Game/Beatmaps/Formats/LegacyBeatmapDecoder.cs

```diff
@@ -62,32 +62,34 @@ protected override void ParseStreamInto(StreamReader stream, Beatmap beatmap)
 
         protected override void ParseLine(Beatmap beatmap, Section section, string line)
         {
+            var strippedLine = StripComments(line);
+
             switch (section)
             {
                 case Section.General:
-                    handleGeneral(line);
+                    handleGeneral(strippedLine);
                     return;
                 case Section.Editor:
-                    handleEditor(line);
+                    handleEditor(strippedLine);
                     return;
                 case Section.Metadata:
                     handleMetadata(line);
                     return;
                 case Section.Difficulty:
-                    handleDifficulty(line);
+                    handleDifficulty(strippedLine);
                     return;
                 case Section.Events:
-                    handleEvent(line);
+                    handleEvent(strippedLine);
                     return;
                 case Section.TimingPoints:
-                    handleTimingPoint(line);
+                    handleTimingPoint(strippedLine);
                     return;
                 case Section.HitObjects:
-                    handleHitObject(line);
+                    handleHitObject(strippedLine);
                     return;
             }
 
-            base.ParseLine(beatmap, section, line);
+            base.ParseLine(beatmap, section, strippedLine);
```

Comment:
Do we really want to pass the stripped line up?

Reply:
that looks like a bad idea.